reflex/.templates/web/utils/state.js

```diff
@@ -273,6 +281,12 @@ export const applyRestEvent = async (event, socket) => {
  * @param socket The socket object to send the event on.
  */
 export const queueEvents = async (events, socket) => {
+
+  // Only proceed if the socket is up, otherwise we throw the event into the void
+  if (!socket) {
```

Comment:
for events like `rx.remove_cookie`, `rx.remove_local_storage` that call `queueEvents(initialEvents(), socket)` after to sync with backend state, this shouldn't be an issue since setting them via reflex requires state anyway. I don't expect any repercussions with this. If there're any, don't hesitate to let me know.